core/services/form/meta/InputOptions.php

```diff
@@ -85,14 +84,16 @@ public function options(): array
 
 
     /**
-     * @param int|float|string $option_value
-     * @param int|float|string $display_text
+     * @param array $option
      */
-    public function addOption($option_value, $display_text): void
+    public function addOption($option): void
     {
-        $option_value = sanitize_key($option_value);
-        if (! isset($this->options[ $option_value ])) {
-            $this->options[ $option_value ] = sanitize_text_field($display_text);
+        if (isset($option['label'], $option['value'])) {
+            $label = sanitize_text_field($option['label']);
+            $value = sanitize_text_field($option['value']);
```

Comment:
```suggestion
            $value = sanitize_key($option['value']);
```

values can't have spaces or other oddities in them

Reply:
Done, thanks